pr-deploy/test/app.test.ts

```diff
@@ -17,115 +17,145 @@
 // mock unzipAndMove dependency before importing '../src/app'
 jest.mock('../src/zipper', () => {
   return {
-    unzipAndMove: jest.fn().mockReturnValue(Promise.resolve('gs://serving-bucket/travisBuildNumber')),
+    unzipAndMove: jest
+      .fn()
+      .mockReturnValue(
+        Promise.resolve('gs://serving-bucket/travisBuildNumber')
+      ),
   };
 });
 
-import {Application} from 'probot';
+process.env.GH_CHECK = 'test-check';
+process.env.GH_OWNER = 'test-owner';
+process.env.GH_REPO = 'test-repo';
```

Comment:
This may or may not be relevant for this app, but in the past I've used `jest-preload.ts` to stub values for environment variables like `test_org`, `__TOKEN__`, etc. that are consumed across tests. Useful if these env vars are stubbed elsewhere

Reply:
Ah makes sense. I'll come back to this if I add another test file.